packages/server/src/routes/grants.js

```diff
@@ -154,6 +154,50 @@ router.get('/exportCSV', requireUser, async (req, res) => {
     res.send(csv);
 });
 
+router.get('/exportCSVRecentActivities', requireUser, async (req, res) => {
```

Comment:
Ideally we can add some tests around this route- let me know if you'd like any assistance or patterns since there might not be much available in the repo.